Resources/Maps/asterisk.yml

```diff
@@ -3543,6 +3547,7 @@ entities:
     - type: MovedGrids
     - type: Broadphase
     - type: OccluderTree
+    - type: LoadedMap
```

Comment:
Whaz this?

Reply:
NM, I guess it's normal

Reply:
I don't actually know, I haven't really done anything different as far as running `mapping 69 /maps/asterisk.yml`

Reply:
All good. All the other maps have it. Just odd that it's getting added here. 🤷‍♂️